app/javascript/application.js

```diff
@@ -88,3 +88,7 @@ window.ace = ace; // Publish ace in global namespace
 // Turbo
 import '@hotwired/turbo-rails';
 import './turbo-migration';
+
+// Import of Rails sprocket assets with minimal changes. After the initial migration these files should
+// be moved to an appropriate place after some adjustments to fit modern Rails JS.
+import 'sprocket_asset_import';
```

Comment:
Do you have a naming convention for JS files? I see snake case and kebab case mixed.